tmt/steps/prepare/install.py

```diff
@@ -238,3 +250,11 @@ def go(self, guest):
             guest.execute(
                 f"{check} || {command} install {options} "
                 f"{packages}{yum_check}")
+
+        # Install debug{info,source} from repos
+        if debuginfo_packages:
+            packages = __prepare_install(debuginfo_packages)
+
+            # make sure debuginfo-install is present on the target system
+            guest.execute(f"{command} install -y /usr/bin/debuginfo-install}")
+            guest.execute(f"{check} || debuginfo-install -y {packages}")
```

Comment:
The `check` part cannot be used here as debuginfo packages have stripped names.

Reply:
Indeed. Removed